MANUAL.txt

```diff
@@ -4151,7 +4151,7 @@ capital letter with a period, by at least two spaces.[^2]
 [^2]:  The point of this rule is to ensure that normal paragraphs
     starting with people's initials, like
 
-        B. Russell was an English philosopher.
+        B. Russell was a Welsh philosopher.
```

Comment:
I wonder if this might be a complicated question. Wikipedia says: "Bertrand Russell was born at Trellech, Monmouthshire, now in Wales, into an aristocratic English family..."  Does being born in Wales make you ethnically Welsh?

What if we say "British"?

Reply:
"British" is fine, though his family roots are both Welsh and English,.